fission-server/Cargo.toml

```diff
@@ -32,15 +32,16 @@ bench = false
 
 [dependencies]
 ansi_term = { version = "0.12", optional = true, default-features = false }
-anyhow = { version = "1.0", features = ["backtrace"] }
+anyhow = { workspace = true }
 async-trait = "0.1"
-axum = { version = "0.6", features = ["headers"] }
+axum = { workspace = true }
 axum-tracing-opentelemetry = { version = "0.10", features = ["otlp"] }
 base64 = "0.21"
 chrono = { version = "0.4", default-features = false, features = ["clock"] }
 config = "0.13"
 console-subscriber = { version = "0.1", default-features = false, features = [ "parking_lot" ], optional = true }
 const_format = "0.2"
+fission-core = { path = "../fission-core" }
```

Comment:
@zeeshanlakhani What workflow do I need to follow so that this works both locally and as crates?

Reply:
So, for a workspace, you'll always take this approach. Though, I'd also set the version along with the path. If you set the version, it's essentially using it as you would releasing the crate.